src/FNA3D_Driver_OpenGL.c

```diff
@@ -367,7 +367,7 @@ static int32_t XNAToGL_TextureInternalFormat[] =
 	GL_RG16F,				/* SurfaceFormat.HalfVector2 */
 	GL_RGBA16F,				/* SurfaceFormat.HalfVector4 */
 	GL_RGBA16F,				/* SurfaceFormat.HdrBlendable */
-	GL_RGBA8,				/* SurfaceFormat.ColorBgraEXT */
+	GL_BGRA,				/* SurfaceFormat.ColorBgraEXT */
```

Comment:
This is a spec violation to my knowledge - it's supposed to be BGRA with RGBA8 as the storage format; AFAIK this is the closest we can get to compliance since ES3 doesn't support BGRA order.